src/Plugins/SimplnxCore/src/SimplnxCore/SurfaceNets/MMCellFlag.cpp

```diff
@@ -136,12 +142,12 @@ void MMCellFlag::set(int32_t cellLabels[8])
   }
 }
 
-unsigned char MMCellFlag::numJunctions() const
-{
-  return m_numJunctions;
-}
+// unsigned char numJunctions() const
+//{
+//   return m_numJunctions;
+// }
 
-MMCellFlag::VertexType MMCellFlag::vertexType()
+VertexType vertexType(unsigned int& m_bitFlag)
 {
   unsigned int vertexTypeBits = (m_bitFlag & m_vertexTypeBits) >> VertexTypeShift;
```

Comment:
`m_vertexTypeBits` is seemingly undefined by the time we arrive here, if it is defined elsewhere as a constant of some sort it really should be changed to the typical `k_` naming schema